Praca_domowa_4/Zad_5_1.py

```diff
@@ -0,0 +1,29 @@
+"""Zaimplementuj iterator zwracający jedynie samogłoski z zadanego ciągu znaków:
+
+Przykładowe użycie:
+for char in Vowels('ala ma kota a kot ma ale'):
+"""
+
+class Vowels:
+    def __init__(self, char: str):
+        self.char = char
+        
+    def __iter__(self):
+        self.number = 0
+        return self
+
+    def __next__(self):
```

Comment:
Już prawie. Zobacz, że jak uruchomisz ten kod to dostaniesz coś takiego:
```
a

a


a


o

a

a


o



a

a

e
```
co oznacza, że oddajesz nie tylko samogłoski, ale także spacje. Spacji być nie może. Także trzeba poprawić.